src/ui/components/PasswordModule/PasswordModule.tsx

```diff
@@ -44,25 +49,43 @@ const PasswordModule = forwardRef<PasswordModuleRef, PasswordModuleProps>(
       setHintValue("");
     };
 
+    const handleClearExisting = () => {
+      setAlertExistingIsOpen(false);
+      handleClearState();
+    };
+
     useImperativeHandle(ref, () => ({
       clearState: handleClearState,
     }));
 
     const handleContinue = async (skipped: boolean) => {
       if (!skipped) {
-        await SecureStorage.set(
-          KeyStoreKeys.APP_OP_PASSWORD,
-          createPasswordValue
+        const currentPassword = await SecureStorage.get(
```

Comment:
Can we move it inside `isModal` condition check?
Like that:
```
if (isModal && authentication.passcodeIsSet  ) {
    const currentPassword = await SecureStorage.get(KeyStoreKeys.APP_OP_PASSWORD);
    setAlertExistingIsOpen(currentPassword === createPasswordValue);
    return;
}
```
=> We can limit unnecessary function calls

Reply:
I've changed this but in a slightly different way. 
`return` needs to be isolated inside that `currentPassword === createPasswordValue` if statement, otherwise it will break things.